projects/sbb-esta/angular-maps/esri-layer-list/src/esri-types/esri-types.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { EsriLoaderService } from 'projects/sbb-esta/angular-maps/esri-config/src/esri-loader.service';
+
+import { EsriLayerListModule } from '../esri-layer-list.module';
+
+@Injectable()
+export class EsriTypesService {
```

Comment:
It is a very bad idea to have different classes with the same name. Please either rename all EsriTypesService classes or refactor the functionality into one class in core.